RevenueCatUI/Purchasing/PurchaseHandler.swift

```diff
@@ -96,13 +106,62 @@ extension PurchaseHandler {
         return result
     }
 
+    func trackPaywallView(_ eventData: PaywallEvent.Data) {
```

Comment:
I'm super late to this, but given that we already use "view" and "paywallView" a bunch because SwiftUI, I'd call this a Paywall Impression event

Reply:
Good idea 👍🏻

Reply:
I'll make a PR renaming this in the code. I'll keep the backend event identifier as "view" unless @joshdholtz is also okay changing that.